tensorflow/python/data/util/convert_test.py

```diff
@@ -18,35 +18,43 @@
 from __future__ import division
 from __future__ import print_function
 
+from absl.testing import parameterized
+
 from tensorflow.python.data.util import convert
+from tensorflow.python.data.kernel_tests import test_base
```

Comment:
imports should be lexicographically sorted, so `tensorflow.python.data.kernel_tests` should come before `tensorflow.python.data.util` ... this comment applies throughout this PR

Reply:
addressed in the commit